packages/expo-asset/package.json

```diff
@@ -38,16 +40,15 @@
   },
   "dependencies": {
     "@react-native/assets-registry": "~0.73.1",
-    "blueimp-md5": "^2.10.0",
     "expo-constants": "~15.4.0",
     "expo-file-system": "~16.0.0",
     "invariant": "^2.2.4",
-    "md5-file": "^3.2.3"
+    "md5-file": "^3.2.3",
+    "expo-module-scripts": "^3.4.0",
+    "expo-modules-core": "^1.11.2"
```

Comment:
Nothing should depend on `expo-modules-core` directly. Add a peer dep on `expo` instead